wdl-gauntlet/src/lib.rs

```diff
@@ -337,8 +337,16 @@ pub async fn gauntlet(args: Args) -> Result<()> {
 
         for message in messages {
             diagnostics.push(config::inner::Diagnostic::new(
-                identifier.to_string(),
+                identifier.clone(),
                 message,
+                config
+                    .inner()
+                    .repositories()
+                    .get(&identifier.repository().clone())
```

Comment:
Is the `clone` call here necessary?

Reply:
Oh sorry, repository already returns a reference, so perhaps just `.get(identifier.repository())`?

Reply:
That seems to work! Thanks.